wmn-data.json

```diff
@@ -6602,6 +6602,17 @@
         "m_code" : 404,
         "known" : ["carlito", "taylor"],
         "cat" : "social"
+      },
+      {
+        "name" : "Statuspage",

```

Comment:
Sorry for not catching this.....in the JSON file we alphabetize the sites by their name. Can you move this entry to the S section?